src/components/modals/SettingsModal.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+
+import { Modal, Button } from 'react-bootstrap';
+import FaShareAlt from 'react-icons/lib/fa/share-alt';
+import FaExternalLink from 'react-icons/lib/fa/external-link';
+
+import { chatHandler } from '../../epics/chatEpics';
+
+const shareLink = window.location.href;
+
+const onDeleteAllMsgs = (e) => {
+  if (window.confirm("Are you sure you want to delete every existing chat message from this chat room? This action cannot be undone.")) {
+    chatHandler.sendDeleteAllMessagesSignalToServer();
+  }
+}
+
+const onCopyShareLink = (e) => {
+  navigator.clipboard.writeText(shareLink);
+}
+
+const SettingsModal = ({
+  isVisible,
+  onCloseModal
+}) => {
+
+  return (
+    <div>
+      <Modal show={isVisible} onHide={onCloseModal}>
+        <Modal.Header closeButton>
+          <h2>Settings</h2>
+        </Modal.Header>
+        <Modal.Body>
+          <hr />
+          <h3>Invite to this Chat</h3>
+          <p>
+            Click to copy a link and invite others to this LeapChat room.
+          </p>
+          <Button onClick={onCopyShareLink} bsStyle="primary">
+            Copy Invite Link <FaShareAlt />
+          </Button>
+          <hr />
+          <h3>Feedback</h3>
+          <p>
+            Do you have feedback or suggestions on how we can improve LeapChat? We're listening!{' '}
+            <a href="https://github.com/cryptag/leapchat/issues" target="_blank">
```

Comment:
@jimmcgaw Please add `rel="nofollow noreferrer noopener"` for privacy reasons (github.com won't be told the user got there from leapchat.org ).

Reply:
Actually I'll add this in a PR branched off of this